packages/screens/FeedNewArticle/components/ArticleContentEditor/utils.ts

```diff
@@ -108,3 +110,11 @@ export const markdownTagStyles: MixedStyleRecord = {
     fontWeight: "500",
   },
 };
+
+// Used to get the same parameters at Article creation and consultation
+export const initializeArticleMd = () => {
+  return markdownit({
+    linkify: true,
+    breaks: true,
+  }).use(emoji);
+};
```

Comment:
Shouldn't this be a `const` instead of a function? I don't think we need to reinitialize the config on every post and it might be ressource intensive

Reply:
Yes, better to load it once, I'll do that

Reply:
https://github.com/TERITORI/teritori-dapp/pull/1451/commits/015b7de126abd35df2ca3220799e97aac4970660